frameworks/Java/quarkus/benchmark_config.json

```diff
@@ -2,8 +2,12 @@
   "framework": "quarkus",
   "tests": [
     {
-      "default": {
+      "hibernate": {
```

Comment:
As of right now, you need a test named `"default"` or the toolset spits up a bunch of warnings. You'll have to change this and rename the `quarkus-hibernate.dockerfile` to `quarkus.dockerfile`

Reply:
Thanks. If I do that, will we still get two entries in the benchmark run or do I actually need to split this off into another entry entirely?

Reply:
Sorry, I don't understand the question. We're just changing the key to the test. It shouldn't change the number of entries or require you split anything off.

Reply:
Just to clarify: this will still end up with two different result lines in the benchmark results, right?

Reply:
Ah, yes. You will have a `quarkus` result which will be the first "default" one, and then a `quarkus-pgclient` result. There will still be two different result lines.

Reply:
OK thanks, done.